static/app/views/settings/projectSourceMaps/sourceMapsList.tsx

```diff
@@ -187,30 +198,119 @@ export function SourceMapsList({location, router, project}: Props) {
         project={project}
         sourceMapUploads={sourceMapUploads}
         isLoading={isPending}
-        emptyMessage={t('No source map uploads found')}
+        query={query}
+        onClearSearch={() => handleSearch('')}
         onDelete={id => {
           deleteSourceMaps({bundleId: id, projectSlug: project.slug});
         }}
+        docsLink={sourceMapsLinks.sourcemaps}
       />
       <Pagination pageLinks={headers?.('Link') ?? ''} />
     </Fragment>
   );
 }
 
+function ReactNativeCallOut() {
+  const [selectedTab, setSelectedTab] = useState('expo');
+
+  return (
+    <div
+      css={css`
+        text-align: left;
+        display: grid;
+        gap: ${space(1)};
+      `}
```

Comment:
why not styled?

Reply:
I have seen both in our code. What has been our preference? can update in a follow-up if needed.